harmonic/ros2_overview.md

```diff
@@ -26,9 +26,6 @@ Please follow the [Install Gazebo and ROS document](ros_installation)
 before starting this tutorial. A working installation of ROS 2 and Gazebo is
 required to go further.
 
-Important: Most of this functionality is only available in ROS 2 Rolling.
-We'll backport it to ROS 2 Jazzy soon.
-
```

Comment:
Let's keep this until we actually make a Jazzy release.

Reply:
Done in commit 0dcd5df.